src/components/api-request.js

```diff
@@ -539,9 +539,9 @@ export default class ApiRequest extends LitElement {
               : html`
                 ${fieldType === 'object'
                   ? html`
-                  <div class="tab-panel row" style="min-height:220px; border-left: 6px solid var(--light-border-color); align-items: stretch;">
+                  <div class="tab-panel row" style="min-height:300px; border-left: 6px solid var(--light-border-color); align-items: stretch;">
```

Comment:
Can you share a bit about what's the reason for this change, maybe a couple of pictures or an example spec where we can trade off on what this size should actually be. Obviously it's hard coded for a reason so figuring out the right value for this is always a challenge.

Reply:
Did you try to use it? it not enough room for Request Body button, and that button move under schema

Reply:
Do you have a sample spec I can use to validate the change?

Reply:
![Screen Shot 2022-09-02 at 9 36 31 AM](https://user-images.githubusercontent.com/28315000/188172863-cc351b3f-fc1f-452d-b29a-5588145640f9.png)

Reply:
technically, it totally not usable

Reply:
Next time I would appreciate a sample spec to validate this. From what I've been able to test it seems to okay.